vast/vast.cpp

```diff
@@ -144,11 +172,11 @@ int main(int argc, char** argv) {
   // Print the configuration file(s) that were loaded.
   if (!cfg.config_file_path.empty())
     cfg.config_files.emplace_back(std::move(cfg.config_file_path));
-  for (auto& path : cfg.config_files)
-    VAST_INFO_ANON("loaded configuration file:", path);
+  for (auto& file : cfg.config_files)
+    VAST_INFO_ANON("loaded configuration file:", file);
   // Print the plugins that were loaded, and errors that occured during loading.
-  for (auto&& path : std::exchange(loaded_plugin_paths, {}))
-    VAST_VERBOSE_ANON("loaded plugin:", std::move(path));
+  for (auto&& file : std::exchange(loaded_plugin_paths, {}))
+    VAST_VERBOSE_ANON("loaded plugin:", std::move(file));
```

Comment:
```suggestion
  for (auto&& file : loaded_plugin_paths)
    VAST_VERBOSE_ANON("loaded plugin:", file);
```

Why not go for a simpler solution if it's possible...aka KISS? It seems like the gains in performance do not justify the added complexity, but I might be wrong. 

Either way, I don't have strong opinions on this issue, but I just want to remind that the *simplest* solution often suffices..

Reply:
While I generally disagree with not using best practices, I do not hold too strong of an opinion on the matter to not simply make the requested change.